intel-retail/rtsf-at-checkout-reference-design
# feat(event-reconciler): migrate Event Reconciler to App SDK 3.0.1

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**75**
closes: #75 

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- Comment out lines 45-93 in `docker-compose.loss-detection.yml`
- Run `make reconciler`
- Run `make run-base` 
- Optional: Run `make run-portainer` and check status of Event Reconciler container
- Optional Run `docker compose -f loss-detection-app/docker-compose.portainer.yml down` to close Portainer
- Run `make clean` to clean up.

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/rtsf-at-checkout-reference-design )
